slurm/morph_router.slurm

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+#SBATCH --job-name=morph_router
+#SBATCH --nodes=1
+#SBATCH --ntasks-per-node=1
+#SBATCH --cpus-per-task=8
+#SBATCH --mem=32G
+#SBATCH --time=24:00:00
+#SBATCH --output=logs/morph_router_%j.out
+#SBATCH --error=logs/morph_router_%j.err
+
+# Load necessary modules
+module load anaconda
```

Comment:
I understand that you were testing this with you own conda env, but for the PR can you copy paste the `e2b_router.slurm` file and include your `python scripts/morph_router.py --port 8001 --max_num_sandboxes 20`. This will only work on our cluster, but at least the slurm files will be consistent across the project.